edx_proctoring/backends/rest.py

```diff
@@ -161,7 +162,10 @@ def register_exam_attempt(self, exam, context):
         payload.pop('attempt_code', False)
         log.debug('Creating exam attempt for %r at %r', exam['external_id'], url)
         response = self.session.post(url, json=payload)
+        if response.status_code != 200:
+            raise BackendProvideCannotRegisterAttempt(response.content)
```

Comment:
We have a log.debug for every 200 response below.  Do we want to log something on the failures? Or do we figure that will be handled higher up?

Reply:
PS: You are missing an "r" from "BackendProvideCannot..."

Reply:
The failures will bubble up and be logged nicely. 

`BackendProvideCannotRegisterAttempt` is actually the name of the exception... I'll change that.